.github/workflows/contracts.yml

```diff
@@ -142,25 +142,30 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: Get upstream packages versions
-        uses: keep-network/ci/actions/upstream-builds-query@v1
+        uses: keep-network/ci/actions/upstream-builds-query@v2
         id: upstream-builds-query
         with:
           upstream-builds: ${{ github.event.inputs.upstream_builds }}
-          query: tbtc-contracts-version = github.com/keep-network/tbtc/solidity#version
+          query: |
+            random-beacon-contracts-version = github.com/keep-network/keep-core/random-beacon#version
+            ecdsa-contracts-version = github.com/keep-network/keep-core/ecdsa#version
 
       - name: Resolve latest contracts
-        run: yarn upgrade @keep-network/tbtc@${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }}
+        run: |
+          yarn upgrade \
+            @keep-network/random-beacon@${{ steps.upstream-builds-query.outputs.random-beacon-contracts-version }} \
+            @keep-network/ecdsa@${{ steps.upstream-builds-query.outputs.ecdsa-contracts-version }} \
+            @keep-network/tbtc@${{ github.event.inputs.environment }}
 
       - name: Configure tenderly
-        if: github.event.inputs.environment == 'ropsten'
         env:
           TENDERLY_TOKEN: ${{ secrets.TENDERLY_TOKEN }}
         run: ./config_tenderly.sh
 
-      - name: Deploy contracts
+      - name: Deploy contract
```

Comment:
Is this change needed?

Reply:
No, that got there by mistake, I'm fixing it as part of https://github.com/keep-network/tbtc-v2/pull/392/files.